src/server/services/userHandler.js

```diff
@@ -8,6 +13,11 @@ function findUserByName(username) {
   });
 }
 
+/**
+ * Return an user information by ID from registered/active Users database.
+ * @param {string} id
+ * @returns {JSON} User information
```

Comment:
this should use `@returns {Promise<JSON>}` to specify that is asynchronous and is a promise

Reply:
Correct, fix commit 26afd6f904edc1dc47e0ffa4a0d3f49f4a882527